docs/controls/access_control/ac-05.md

```diff
@@ -0,0 +1,53 @@
+# Access Control
+## AC-05 - Separation of Duties
+
+The organization:  
+a. Separates [at a minimum, data creation and control, software development and maintenance, and security functions];
+
+b. Documents separation of duties of individuals; and 
+
+c. Defines information system access authorizations to support separation of duties.
+
+### TDP Implementation
+
+**Part a.**  
+The TANF Data Portal (TDP) system has explicitly defined the roles and responsibilities that accounts perform to operate and maintain the system. This includes data creation and processing, software development, maintenance, and security implementation.
+
+Every role has flags associated with the account.  These flags are *SuperUser* and *Staff* and are set to *False* by default.  When a new TDP account is created, the OFA Admin has the ability to set the flags to *True* based on the user's job responsibilities.  Note: if an account has the *Staff* flag set to *True*, the account will have access to Django Admin, but will not have access to any of the account management functionality.
```

Comment:
Should `OFA Admin` be `System Owner` in this paragraph? Otherwise this statement implies that all OFA Admin's can set the Superuser and Staff flags, which doesn't seem like the case based on the following paragraph.

Reply:
Yes, `OFA Admin` does not have the ability to set those flags. Only the `System Owner`.

Reply:
Updated to System Owner

Reply:
Removed system owner and changed to system admin